framework/lib/theme/components/tag/index.ts

```diff
@@ -28,16 +25,18 @@ export const Tag: ComponentMultiStyleConfig = {
     },
   },
   variants: {
-    solid: ({ bgColor, colorScheme }) => ({
+    solid: ({ theme: { colors }, colorScheme }) => ({
       container: {
-        bgColor: bgColor ?? colorScheme,
-        color: getContrastColor(bgColor ?? useToken('colors', colorScheme)),
+        bgColor: colors[colorScheme] && colors[colorScheme][500]
+          ? colors[colorScheme][500] : colorScheme,
+        color: 'white',
```

Comment:
Small change here , to be 1:1 with the design, you would need to add a simple rule re to be able to achieve that. As you can see in the [Design File](https://www.figma.com/file/0rnkmzpBhBoVl3PsbZDmGD/%E2%AC%9B%EF%B8%8F%C2%A0%C2%A0Core.mainCOMPONENTS?type=design&node-id=2-28&mode=design)

The yellow badge has a black color. Since there is no requested change for theming here, that color stays the same.

Let's add this rule in this line to make it right `color: colorScheme === 'yellow' ? 'mono.black' : 'mono.white',`